src/Check/PhpVersion.php

```diff
@@ -23,64 +18,47 @@
  *
  * This test accepts a single version and an operator or an array of
  * versions to test for.
+ *
+ * @psalm-type Operator = '<'|'lt'|'<='|'le'|'>'|'gt'|'>='|'ge'|'=='|'='|'eq'|'!='|'<>'|'ne'
```

Comment:
```suggestion
 * @psalm-type Operator = value-of<self::VALID_OPERATORS>
```

The result would be the same and this could be reused later on?

Reply:
True that :+1: